packages/agoric-cli/src/lib/format.js

```diff
@@ -0,0 +1,74 @@
+// @ts-check
+
+// ambient types
+import '@agoric/ertp/src/types.js';
+
+export const COSMOS_UNIT = 1_000_000n;
+
+/**
+ * Like @endo/nat but coerces
+ *
+ * @param {string} str
+ * @returns {bigint}
+ */
+export const Natural = str => {
+  const b = BigInt(str);
+  if (b < 0) {
+    throw new RangeError(`${b} is negative`);
+  }
+  return b;
+};
+
+// eslint-disable-next-line no-unused-vars
+const exampleAsset = {
+  brand: { boardId: 'board0425', iface: 'Alleged: BLD brand' },
+  displayInfo: { assetKind: 'nat', decimalPlaces: 6 },
+  issuer: { boardId: null, iface: undefined },
+  petname: 'Agoric staking token',
+};
+/** @typedef {typeof exampleAsset} AssetDescriptor */
+
+/** @param {AssetDescriptor[]} assets */
+export const makeAmountFormatter = assets => amt => {
+  const {
+    brand: { boardId },
+    value,
+  } = amt;
+  const asset = assets.find(a => a.brand.boardId === boardId);
+  if (!asset) return [NaN, boardId];
+  const {
+    petname,
+    displayInfo: { assetKind, decimalPlaces },
+  } = asset;
+  if (assetKind !== 'nat') return [['?'], petname];
+  /** @type {[qty: number, petname: string]} */
+  const scaled = [Number(value) / 10 ** decimalPlaces, petname];
```

Comment:
I was aiming or `[5, "IST"]` here; I got it in an earlier version of `psm-tool`, but using `petname` here gives us `[5,"Agoric stable local currency"]`. I think the `"IST"` name is called `issuerName` somewhere in the wallet world. Here's hoping we integrate that in due course. (not critical).